src/features/adr/pages/PatientForm.tsx

```diff
@@ -33,19 +34,78 @@ const useStyles = stylesFactory({
 export interface PatientFormProps {
   onSubmit: (data: any) => void;
   step: number;
+  jsonSchema: any;
+  patientName?: string;
+  setJsonSchema: any;
 }
 
-export const PatientForm: FC<PatientFormProps> = ({ onSubmit, step }) => {
+const mapHistory = (history: PatientHistory[]) =>
+  history
+    .filter((h) => h.transLines.some((t) => t.itemLine.item.is_vaccine))
```

Comment:
Not sure if this is used in Tonga at all and I'm almost certain it is a 'their data issue' but if I lookup a certain patient using Tongas datafile, I get an error

![image](https://user-images.githubusercontent.com/35858975/126429664-dbb8bad8-7f0a-4a2e-aac4-d72f519ab89f.png)

As there are trans lines with no item lines 🙄

Reply:
It is 'their data' but we should handle that. so we will..

Reply:
8ab16fd